src/index.html

```diff
@@ -6,7 +6,136 @@
     <title>Wbpack Exercise</title>
   </head>
   <body>
-    <h1>Hello webpack!</h1>
-    <h2>this is test again right now?</h2>
+
+
+    <main id="main-section">
+      <!-- cardboards -->
+      <ul id="cardboards">
+
+        <li id="cardboard">
+          <img id="cardboard-image" class="cardboard-image" alt="img">
+          <div id="cardboard-superior">
+            <h3 id="cardboard-title" class="cardboard-title">Space 1</h3>
+            <div id="cardboard-inferior">
+              <img id="cardboard-heart-image" class="cardboard-heart-image" alt="img">
+              <p id="cardboard-likes">0 likes</p>
+            </div>
+          </div>
+          <button id="comments-button">Comments</button>
+          <button id="reservations-button">Reservations</button>
+        </li>
+
+        <li id="cardboard">
+          <img id="cardboard-image" class="cardboard-image" alt="img">
+          <div id="cardboard-superior">
+            <h3 id="cardboard-title" class="cardboard-title">Space 1</h3>
+            <div id="cardboard-inferior">
+              <img id="cardboard-heart-image" class="cardboard-heart-image" alt="img">
+              <p id="cardboard-likes">0 likes</p>
+            </div>
+          </div>
+          <button id="comments-button">Comments</button>
+          <button id="reservations-button">Reservations</button>
+        </li>
+
+        <li id="cardboard">
+          <img id="cardboard-image" class="cardboard-image" alt="img">
+          <div id="cardboard-superior">
+            <h3 id="cardboard-title" class="cardboard-title">Space 1</h3>
+            <div id="cardboard-inferior">
+              <img id="cardboard-heart-image" class="cardboard-heart-image" alt="img">
```

Comment:
I understand what you did here, 🤔 I think adding each cardboard manually is going to cause some problems in the future when we might want to update the styles or add new classes.
#### What if we create these items programmatically? that way we can even add event listeners to the buttons when each cardboard is created